### DISCUSSION TITLE
Dailymotion

### DISCUSSION THREAD

**Discussion by southerndream56** [iptv-org/community#536] [Q&A] [answered]
Hello everybody,
Does anybody knows how to enable Dailymotion link to work stable as m3u line?
Because they change link parameters quite often.
For example : https://stream-05.ix7.dailymotion.com/sec(p5KA4QqcEoMI-Au0U41kOmEmkdGyVsgcnRuJm4Ss9vSY8MkQuVOyjAHQyyrA6tas)/dm/3/x84eirw/s/live-2.m3u8
Thanks for all your help

**Comment by Dum4G** [comment, marked as answer]
You need to use a player that can properly query Dailymotion links or host yourself a utillity that will do that somewhere remotely via Streamlink
https://github.com/BellezaEmporium/query-streamlink

**Reply by southerndream56** [reply]
Thanks for your answer.
Can you suggest me a player that can query such links?

**Reply by Dum4G** [reply]
VLC does that. You can try something else from https://github.com/iptv-org/awesome-iptv

**Reply by southerndream56** [reply]
Thanks,
I'm not sure if I explained my problem properly...
I am able to find m3u line while watching Dailymotion LIVE channel with my browser,but when I put this line
to VLC player or Smart TV, it changes every ,let's say , 24h.
Is there a procedure how to change this m3u line to work constantly ?

**Reply by Dum4G** [reply]
m3u line itself will change every 24h, nothing can be done about that. You can only find a player that will scrap that m3u from the video page or automate the process by hosting yourself an app that will update that link on it's own and return it to you by other means. For example on your own domain, somewhere on free host like Heroku or via github Actions where m3u will be reuploaded as a file containing the right link for today. The choise is to do it clientside if the player allows it or via automation tools that need to be hosted somewhere.
